Comparing source compatibility of /home/jenkins/.m2/repository/org/apache/jspwiki/jspwiki-cache/3.0.1-SNAPSHOT/jspwiki-cache-3.0.1-SNAPSHOT.jar against /home/jenkins/.m2/repository/org/apache/jspwiki/jspwiki-cache/2.12.3/jspwiki-cache-2.12.3.jar
WARNING: You are using the option '--ignore-missing-classes', i.e. superclasses and interfaces that could not be found on the classpath are ignored. Hence changes caused by these superclasses and interfaces are not reflected in the output.
===  UNCHANGED CLASS: PUBLIC org.apache.wiki.cache.CacheInfo  (not serializable)
	***! CLASS FILE FORMAT VERSION: 61.0 <- 55.0
	===  UNCHANGED SUPERCLASS: java.lang.Object (<- java.lang.Object)
	===  UNCHANGED CONSTRUCTOR: PUBLIC CacheInfo(java.lang.String, long)
	===  UNCHANGED METHOD: PUBLIC long getHits()
	===  UNCHANGED METHOD: PUBLIC long getMaxElementsAllowed()
	===  UNCHANGED METHOD: PUBLIC long getMisses()
	===  UNCHANGED METHOD: PUBLIC java.lang.String getName()
	===  UNCHANGED METHOD: PUBLIC void hit()
	===  UNCHANGED METHOD: PUBLIC void miss()
===  UNCHANGED INTERFACE: PUBLIC ABSTRACT org.apache.wiki.cache.CachingManager  (not serializable)
	***! CLASS FILE FORMAT VERSION: 61.0 <- 55.0
	===  UNCHANGED SUPERCLASS: java.lang.Object (<- java.lang.Object)
	===  UNCHANGED FIELD: PUBLIC STATIC FINAL java.lang.String CACHE_ATTACHMENTS_DYNAMIC
	===  UNCHANGED FIELD: PUBLIC STATIC FINAL java.lang.String CACHE_ATTACHMENTS
	===  UNCHANGED FIELD: PUBLIC STATIC FINAL java.lang.String CACHE_DOCUMENTS
	===  UNCHANGED FIELD: PUBLIC STATIC FINAL java.lang.String CACHE_PAGES_TEXT
	===  UNCHANGED FIELD: PUBLIC STATIC FINAL java.lang.String PROP_USECACHE_DEPRECATED
		===  UNCHANGED ANNOTATION: java.lang.Deprecated
	===  UNCHANGED FIELD: PUBLIC STATIC FINAL java.lang.String CACHE_PAGES
	===  UNCHANGED FIELD: PUBLIC STATIC FINAL java.lang.String CACHE_ATTACHMENTS_COLLECTION
	===  UNCHANGED FIELD: PUBLIC STATIC FINAL java.lang.String CACHE_PAGES_HISTORY
	===  UNCHANGED FIELD: PUBLIC STATIC FINAL java.lang.String PROP_CACHE_CONF_FILE
	===  UNCHANGED FIELD: PUBLIC STATIC FINAL java.lang.String PROP_CACHE_ENABLE
	===  UNCHANGED METHOD: PUBLIC ABSTRACT boolean enabled(java.lang.String)
	===  UNCHANGED METHOD: PUBLIC ABSTRACT java.lang.Object get(java.lang.String, java.io.Serializable, org.apache.wiki.util.CheckedSupplier<T,E><T,E>)
		===  UNCHANGED EXCEPTION: java.lang.Exception
		GENERIC TEMPLATES: === E:java.lang.Exception, === T:java.lang.Object
	===  UNCHANGED METHOD: PUBLIC ABSTRACT org.apache.wiki.cache.CacheInfo info(java.lang.String)
	===  UNCHANGED METHOD: PUBLIC ABSTRACT java.util.List<T><T> keys(java.lang.String)
		GENERIC TEMPLATES: === T:java.io.Serializable
	===  UNCHANGED METHOD: PUBLIC ABSTRACT void put(java.lang.String, java.io.Serializable, java.lang.Object)
	===  UNCHANGED METHOD: PUBLIC boolean registerListener(java.lang.String, java.lang.String, java.lang.Object[])
	===  UNCHANGED METHOD: PUBLIC ABSTRACT void remove(java.lang.String, java.io.Serializable)
	===  UNCHANGED METHOD: PUBLIC ABSTRACT void shutdown()
===  UNCHANGED CLASS: PUBLIC org.apache.wiki.cache.EhcacheCachingManager  (not serializable)
	***! CLASS FILE FORMAT VERSION: 61.0 <- 55.0
	===  UNCHANGED SUPERCLASS: java.lang.Object (<- java.lang.Object)
	===  UNCHANGED CONSTRUCTOR: PUBLIC EhcacheCachingManager()
	===  UNCHANGED METHOD: PUBLIC boolean enabled(java.lang.String)
	===  UNCHANGED METHOD: PUBLIC java.lang.Object get(java.lang.String, java.io.Serializable, org.apache.wiki.util.CheckedSupplier<T,E><T,E>)
		===  UNCHANGED EXCEPTION: java.lang.Exception
		GENERIC TEMPLATES: === E:java.lang.Exception, === T:java.lang.Object
	===  UNCHANGED METHOD: PUBLIC org.apache.wiki.cache.CacheInfo info(java.lang.String)
	===  UNCHANGED METHOD: PUBLIC void initialize(org.apache.wiki.api.core.Engine, java.util.Properties)
		===  UNCHANGED EXCEPTION: org.apache.wiki.api.exceptions.WikiException
	===  UNCHANGED METHOD: PUBLIC java.util.List<java.lang.String><java.lang.String> keys(java.lang.String)
	===  UNCHANGED METHOD: PUBLIC void put(java.lang.String, java.io.Serializable, java.lang.Object)
	===  UNCHANGED METHOD: PUBLIC boolean registerListener(java.lang.String, java.lang.String, java.lang.Object[])
	===  UNCHANGED METHOD: PUBLIC void remove(java.lang.String, java.io.Serializable)
	===  UNCHANGED METHOD: PUBLIC void shutdown()

Semantic versioning suggestion: 0.0.1